### ISSUE TITLE
Enhance Shaman Hit Talent Not Applying

### ISSUE THREAD

**Issue by Davud-P** [gamefreedomgit/Frostmourne#311]
**Description:**
Enhancement Shaman's 'Dual Wield Mastery' hit bonus does not properly apply to spell hit.

**How it should work:**
The hit bonus should apply to both white-hits and spell-hit.

**Comment by Nihaobeachez** [comment]
You sure it should apply to spell-hit? All sources ive found is that it doesnt affect it.

https://youtu.be/fsizOsYyAoY?si=a3TYD97MEREUAHOH&t=433 

![image](https://github.com/gamefreedomgit/Frostmourne/assets/96308500/c85382d7-0765-44f5-a1d0-43f6359e745a)

**Comment by Davud-P** [comment]
You are correct. I did some deeper digging, and the wording of the talent is (and has been) misleading for 16 years. While other talents specify Spell Hit or Melee Hit, Shamans talent simply specifies "Chance to Hit."

I also found this older thread. I will close this issue, as I was mistaken.

![Screenshot_20240330-103606](https://github.com/gamefreedomgit/Frostmourne/assets/12061473/527f0193-aec5-4839-a9a8-09833a52708d)
